src/components/ToDoComponent/TodoComponent.tsx

```diff
@@ -6,14 +6,20 @@ import { TodoStyles } from './Styles/TodoStyles';
 import Task from './taskType';
 
 const TodoComponent = ({ navigationProp }) => {
-  const [tasks, setTasks] = useState<Task[]>(data);
+  const [tasks, setTasks] = useState<Task[]>([]);
   const [disabled, setDisabled] = useState<boolean>(false);
   const [changed, setChanged] = useState<boolean>(false);
 
   useEffect(() => {
-    setTasks(tasks);
+    getTodos()
   }, [changed, tasks]);
 
+  const getTodos = async() => {
+    const todos = await fetch('https://backend-goals-production.up.railway.app/goal/');
```

Comment:
Can you create the constants for this API URL fetch is default function or we created using axios or other library ?

Reply:
already done in previous PR will take a pull of the same and update it

Reply:
Ok